binding/python/generate.py

```diff
@@ -472,11 +472,11 @@ def build_qp(tasks):
                       'tasks::qp::JointGains', 'vector')
   tasks.add_container('std::vector<tasks::qp::SpringJoint>',
                       'tasks::qp::SpringJoint', 'vector')
-  tasks.add_container('std::vector<Eigen::Vector3d>', 'Eigen::Vector3d', 'vector')
-  tasks.add_container('std::vector<Eigen::Matrix3d>', 'Eigen::Matrix3d', 'vector')
-  tasks.add_container('std::vector<Eigen::VectorXd>', 'Eigen::VectorXd', 'vector')
-  tasks.add_container('std::vector<std::vector<Eigen::VectorXd> >',
-                      'std::vector<Eigen::VectorXd>', 'vector')
+  tasks.add_container('std::vector<Eigen::Vector3d, Eigen::aligned_allocator<Eigen::Vector3d> >', 'Eigen::Vector3d', 'vector')
+  tasks.add_container('std::vector<Eigen::Matrix3d, Eigen::aligned_allocator<Eigen::Matrix3d> >', 'Eigen::Matrix3d', 'vector')
+  tasks.add_container('std::vector<Eigen::VectorXd, Eigen::aligned_allocator<Eigen::VectorXd> >', 'Eigen::VectorXd', 'vector')
```

Comment:
This is *wrong*, you should *not* use aligned allocation for dynamic size types... Even for some fixed-size objects, such care is not necessary. See here:
http://eigen.tuxfamily.org/dox-devel/group__TopicFixedSizeVectorizable.html

Long story short: things that have an even number of doubles in them (Vector2d, Vector4d, Matrix2d...).